pandas/tests/groupby/transform/test_transform.py

```diff
@@ -1490,3 +1490,41 @@ def test_idxmin_idxmax_transform_args(how, skipna, numeric_only):
         msg = f"DataFrameGroupBy.{how} with skipna=False encountered an NA value"
         with pytest.raises(ValueError, match=msg):
             gb.transform(how, skipna, numeric_only)
+
+
+def test_transform_sum_with_series():
+    # One column
+    df = DataFrame({"X": [1.0]})
+    series = Series(["Y"])
+    result = df.groupby(series, as_index=False).transform("sum")
+    expected = DataFrame({"X": [1.0]})
+    tm.assert_frame_equal(result, expected)
+
+    # With labels
```

Comment:
Can you turn these comments into separate tests instead?

Reply:
ok, done